docs/LearnCalico.md

```diff
@@ -0,0 +1,364 @@
+
+Felix also programs interface information to the kernel for outgoing endpoint 
+traffic. Felix instructs the host to respond to ARPs for workloads with the 
+MAC address of the host.
+
+#### BIRD internet routing daemon
+
+BIRD is an open source BGP client that is used to exchange routing information 
+between hosts.  The routes that Felix programs into the kernel for endpoints 
+are picked up by BIRD and distributed to BGP peers on the network.
+
+#### confd templating engine 
+
+The confd templating engine watches the etcd datastore for any changes to BGP 
+configuration.  Confd dynamically generates BIRD configuration files based on 
+these changes, then triggers BIRD to load the new files.
+
+## Lifecycle of a Calico Networked Container
+
+Below is an example of what happens behind the scenes when you start Calico on 
+two hosts, create and add containers to Calico, and configure endpoints with 
+security policy for each container.  This example utilizes Docker default 
+networking, which differs in some ways from using Calico with the Docker 
+libnetwork plugin and other orchestrators like Kubernetes, Mesos, and rkt.
+
+### Start the calico/node instance on your container hosts
+
+With two hosts on the same network, on each host, the `calicoctl node` command 
+is run.  This creates a container called `calico-node`, which uses the 
+`calico/node` Docker image.
+
+At this point we have a Felix agent, a BIRD BGP daemon, and confd running in the 
+calico-node container on each host. The `calicoctl` command stores the default 
+global AS number and the IP address of each host as BGP host addresses in etcd. 
+The confd process reads these BGP IP addresses and uses a template to configure 
+the BGP daemon with BGP peers.  The two hosts are BGP peers connected in a full 
+mesh using the default global AS number.
+
+There are currently no workload containers running and Felix is standing by, 
+waiting for changes in the etcd datastore.
+
+![calicoctl node](diagrams/calicoctl_node.png)
+
+calico-docker allows you to configure other BGP topologies as well, such as a 
+mesh of route reflectors or per-node BGP.  Visit the calico-docker [BGP 
+page](https://github.com/projectcalico/calico-docker/blob/master/docs/bgp.md) 
+for more details.
+
+### Create Containers on Host
+
+Two containers are created using the orchestrator (such as running
+`docker run --name=workload_A -tid ubuntu`).  The workloads have not been added 
+to Calico networking and no security policy has been configured at this point, 
+so they do not yet have connectivity or Calico-assigned IP addresses.  
+
+The workloads do not have connectivity between one another, but they can access 
+the internet via the Docker bridge interface. None of the Calico processes have 
+performed any new actions at this point.
+
+![docker run](diagrams/docker_run.png)
+
+**Note**: Docker's libnetwork plugin works a bit differently than this 
```

Comment:
Note sure if we should document deltas or just have different workflows for the different scenarios.